package.json

```diff
@@ -3,10 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev -p build-fragment",
-    "build": "next build -p build-fragment",
-    "start": "next start -p build-fragment",
-    "build-fragment": "node ./schemaQuery.ts"
+    "dev": "npm-run-all -p build-fragment dev-next",
+    "dev-next": "next dev",
```

Comment:
@MonikaCat I have created a new script, then use the original `dev` script to run both scripts with `npm-run-all` in parallel.